newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js

```diff
@@ -149,13 +150,24 @@ export default class LocalPreviewLauncher extends React.Component<
         new gd.AbstractFileSystemJS(),
         localFileSystem
       );
-      const outputDir = path.join(fileSystem.getTempDir(), 'preview');
-      const exporter = new gd.Exporter(fileSystem, gdjsRoot);
-
-      return {
-        outputDir,
-        exporter,
-      };
+      try {
```

Comment:
You don't need to do any changes in this file, because the uid is already handled in the file system.

Reply:
Oh... Ok...

Reply:
I know it's long, but you're almost done :)